### ISSUE TITLE
Latest Release/snapshot profile

### ISSUE THREAD

**Issue by distrair** [PrismLauncher/PrismLauncher#550]
### Role

Much better version of official launcher but not written on Electrom

### Suggestion

I want to have ability to always have the latest version of Minecraft

### Benefit

For people who wants to have a always up-to-date version of Minecraft

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

**Comment by Bjorkan** [comment]
Hi! I would like to add some reasoning to further support this request!

## 1. Smoother transition for beginners
When introducing other people to a third party launcher, many people are scared that they will not understand it or that it will be too complicated. By throwing everything at them at the same time, with separate folders for all instances (which is good, don't get me wrong), new looking launcher, and many options, a default instance not containing anything more than the latest release of Minecraft would soften peoples transition over to Prism Launcher, and over time they can explore all that the launcher offer!

![image](https://github.com/PrismLauncher/PrismLauncher/assets/93771679/0e9aecc2-73cb-4619-bdb5-afc738a9995b)


## 2. Realms
I personally don't use Realms as I think hosting is a better alternative, you cannot deny the simplicity of paying Microsoft a small amount and them hosting your world for you and your friends. However as Realms always updates to the latest version of Minecraft right after the release of a new version, an instance dedicated to just staying on the latest version would make the experience extremely better for those players!

## 3. Latest Snapshot Instance?
The official Microsoft Launcher also includes an option for the latest Snapshot, something if any is an amazing option if brought over to Prism Launcher. Snapshots are very rarely played on for longer periods of times and often one only wants to try the latest and greatest Minecraft has to offer. Therefore the lack of mod support for an instance like this, is not an issue and would greatly enhance the experience using the launcher.

## Conclusion
I understand why this option is easily overlooked as not needed because many modpacks greatly enhance the experience and that most players will just go that route, but for those just dipping their toes into the amazing world of Minecraft modding, the familarity of the Latest release and Latest Snapshot instance will be a great stepping stone to help them explore the world of modding at their own pace, and makes them move over to Prism Launcher for good.

Thank you and I'll gladly respond to messages below if you have any!

Wish you all the best,
Jesper

**Comment by TheKodeToad** [comment]
Did you use ChatGPT?

**Comment by Bjorkan** [comment]
> Did you use ChatGPT?

No I did not haha! Was it that bad? 😅 I'm sorry in that case 😂

**Comment by clammish** [comment]
I fully support this request, Prism is just so much better than the default launcher in so many ways but this is one of the few things it's missing.

**Comment by bentobaku** [comment]
4yrs, How is this still not added, this is such an issue for me. If Linux had a standard launcher, I would rather use it x100 more just because of this issue. Its stupid you still haven't done something about it. I only play vanilla, and when I am on Linux, I'm forced to use prism because Microsoft is Microsoft. I am stuck with a beautiful, over powered, launcher that I have to constantly delete and remake an instance just to keep playing. In the past this wasn't that much of an issue. But in modern times MC updates constantly and this is becoming more and more annoying to deal with. If auto-updating is too much, maybe just add an right-click option on the instance to update it without having to run through multiple steps.

**Comment by Bjorkan** [comment]
> 4yrs, How is this still not added, this is such an issue for me. If Linux had a standard launcher, I would rather use it x100 more just because of this issue. Its stupid you still haven't done something about it. I only play vanilla, and when I am on Linux, I'm forced to use prism because Microsoft is Microsoft. I am stuck with a beautiful, over powered, launcher that I have to constantly delete and remake an instance just to keep playing. In the past this wasn't that much of an issue. But in modern times MC updates constantly and this is becoming more and more annoying to deal with. If auto-updating is too much, maybe just add an right-click option on the instance to update it without having to run through multiple steps.

Uh you don't need to rempve your instance to update it. It's just a button in settings to do it😅

**Comment by bentobaku** [comment]
Yeah, I finally found it, still way too many step for someone just playing vanilla MC.
